app/library/postgres.py

```diff
@@ -149,6 +149,13 @@ async def delete_asset_from_postgres(asset_id: str, session):
             Key=asset_obj.s3_key,
         )
 
+        pos_key = asset_obj.screenshot_url.find("image/")
```

Comment:
Ah, forgot one thing, some images won't have screenshot URL, so this will be an error. I would suggest nesting the rest of this under an if statement like `if pos_key != -1`

Reply:
ooo thanks for catching that, updated